apps/oozie/src/oozie/forms.py

```diff
@@ -43,24 +43,24 @@ class ParameterForm(forms.Form):
   value = forms.CharField(max_length=12288, required=False)
 
   NON_PARAMETERS = (
-      'user.name',
-      'mapreduce.job.user.name',
-      'wf_application_path',
-      'jobTracker',
-      'nameNode',
       'hue-id-w',
       'hue-id-c',
       'hue-id-b',
       'hue-id-b',
       'security_enabled',
-      'oozie.wf.rerun.failnodes',
-      'dryrun',
-      'send_email'
+      'dryrun'
   )
 
   RERUN_HIDE_PARAMETERS = (
       'security_enabled',
-      'dryrun'
+      'dryrun',
+      'user.name',
+      'mapreduce.job.user.name',
+      'wf_application_path',
+      'jobTracker',
+      'nameNode',
+      'oozie.wf.rerun.failnodes',
```

Comment:
Are you sure it does not break the re-run of a failed workflow?

Except this nice!

Reply:
Sorry. I'm not sure. Just test success in Coordinator job.  I will check it.

Reply:
I check it. It's working nicely.